test/tests/functional/pbs_test_run_count.py

```diff
@@ -53,7 +54,23 @@ def create_reject_begin_hook(self):
 
         # make sure hook has propogated to mom
         self.mom.log_match("h1.HK;copy hook-related file request received",
-                           existence=True)
+                           existence=True, starttime=start_time)
+
+    def check_run_count(self, input_count="0", output_count="1"):
```

Comment:
@tpai26 I'd suggest you change the default output_count to 21 because it is only when it reaches 21 that the job goes into HELD state.

Reply:
done